client/error.go

```diff
@@ -64,9 +63,7 @@ func ErrorClassifier(meta schema.ClientMeta, resourceName string, err error) []d
 	if ok := errors.As(err, &gerr); ok {
 		if grpcCode, ok := httpCodeToGRPCCode[gerr.Code]; ok {
 			if v, ok := grpcCodeToDiag[grpcCode]; ok {
-				return []diag.Diagnostic{
-					diag.FromError(err, v.severity, v.typ, resourceName, v.summary, err.Error()),
-				}
+				return execution.FromError(err, execution.WithSeverity(v.severity), execution.WithType(v.typ), execution.WithResource(resourceName), execution.WithSummary(v.summary), execution.WithDetails(err.Error()))
```

Comment:
Not sure about the `execution.WithDetails(err.Error())` part...

Reply:
Yes, I agree, this might include user related info.

Reply:
Fixed